### PR TITLE
[REVIEW] ENH Add dask-cuda docs

### PR DESCRIPTION
**Pull request by dillon-cullinan**
Adds dask-cuda docs to the RAPIDS docs site.

### CODE REVIEW

File: _data/docs.yml [hunk 1]
```diff
@@ -101,6 +101,17 @@ apis:
       legacy: 0
       stable: 0
       nightly: 1
+  cucim:
```
Comment (charlesbluca) [inline _data/docs.yml:104]:
```suggestion
  dask-cuda:
```

**Review by charlesbluca — CHANGES_REQUESTED** [review]
Thanks @dillon-cullinan! I'll include @rapidsai/daskcuda-python-codeowners for viz; not sure if there are any thoughts on what the blurb should say

**Comment by dillon-cullinan** [comment]
Ya I just copied it from the top of the Dask CUDA README, figured that would be fine.

**Review by charlesbluca — APPROVED** [review]
Thanks again @dillon-cullinan!

**Review by ajschmidt8 — APPROVED** [review]

**Pull request merged**